src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -268,9 +268,6 @@ internal override void PrepareForRecycle()
 			// ListView to get a clicked container.
 			ClearValue(ItemsControl.IndexForItemContainerProperty);
 
-			// Reset visual state so that the container doesn't come back looking like it's hovered or clicked.
```

Comment:
@dr1rrb isn't this something you were looking into for pointers?

Reply:
No, my issue was even without recycling.